### PR TITLE
Prmotions and delete draft

### PR DESCRIPTION
**Pull request by Bsaint95**

### CODE REVIEW

**Comment by Bsaint95** [comment]
Capture
Capture

**Comment by Bsaint95** [comment]
Capture
Capture
Capture

File: businessManagement/views.py [hunk 1]
```diff
@@ -139,8 +139,8 @@ def get_all_receipt(request):
             return JsonResponse({"message": error}, status=status.HTTP_400_BAD_REQUEST)
 
 
-@api_view(["GET"])
-def get_all_draft_receipt(request):
+@api_view(["GET", "DELETE"])
+def get_all_draft_receipt(request, id):
```
Comment (poojithamiryala) [inline businessManagement/views.py]: Without adding id in the urls param how will you be able to fetch

File: promotions/urls.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
 from django.urls import path
-from .views import create_promotion, update_promotion, delete_promotion
+from .views import (
+	create_promotion, 
+	update_promotion, 
+	delete_promotion)
 
 urlpatterns = [
-    path("promotion/create", create_promotion),
```
Comment (poojithamiryala) [inline promotions/urls.py:5]: Can remove this changes

File: promotions/views.py [hunk 3]
```diff
@@ -5,12 +5,10 @@
...
 from django.http import JsonResponse
 
-# Create your views here.
-#  delete,update
 
 
 @api_view(["POST"])
-def add_promotion(request):
+def create_promotion(request):
     if request.method == "POST":
```
Comment (poojithamiryala) [inline promotions/views.py:13]: Can also remove changes in this file

**Review by poojithamiryala — CHANGES_REQUESTED** [review]

**Pull request merged**